### PR TITLE
Add Toolbar and headline

### PR DESCRIPTION
**Pull request by Ahmed-Alfaouq**
What I did:
1) Setup environment
2) Add Toolbar and its style
3) Add Headline section and its style
4) Fix some stylelint errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        </div>
+        <div class="collapse-menu">
+          <ul>
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+            <li>Letter Icon</li>
+          </ul>
+        </div>
+    </nav>
```
Comment (AshakaE) [inline index.html:28]: The toolbar is part of the header section

- [ ] Kindly replace the `nav` element with a `header` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="It's my portfolio. Here you will find my projects and experience Take a look now!">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+  <!-- Start Navbar -->
+    <nav>
+        <div class="logo">Ahmed Omar</div>
```
Comment (AshakaE) [inline index.html:14]: - [ ] Kindly use an `a` tag for the logo, this will serve as a home button for the webpage

File: index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        <div class="menu-icons">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
```
Comment (AshakaE) [inline index.html:19]: The menu icon is the actual navigation for the webpage

- [ ] Kindly replace the `div` element with a `nav` tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+            <li>Letter Icon</li>
+          </ul>
+        </div>
+    </nav>
+  <!-- End Navbar -->
+  <!-- Start Headline -->
+    <section class="headline">
```
Comment (AshakaE) [inline index.html:31]: - [ ] Kindly add the appropriate background image for this section 🙏

File: index.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+      <h1>Hey There. I’m Ahmed</h1>
+      <h2>I’am a Software Developer</h2>
+      <p>
+        I can help you build a product, feature or <br>
+        website Look through some of my work <br>
+        and experience! If you like what you see <br>
+        and have a project you need coded, <br> 
+        don’t hestiate to contact me.
+      </p>
+      <div class="social-media">
```
Comment (AshakaE) [inline index.html:41]: - [ ] Kindly use a `ul` tag to wrap all the social icons

File: index.html [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        <div class="image-container">
+          <img src="/images/github.png" alt="github">
+        </div>
```
Comment (AshakaE) [inline index.html:44]: - [ ] Kindly use an `li` tag per social icon and an `a` tag for the icon itself. you can wrap the `img` tag with an `a` tag 💡

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Ahmed-Alfaouq  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project 🛠️   but you are almost there! 

#  Required Changes ♻️ 🚧
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Ahmed-Alfaouq) [reply index.html:19]: sir,
it's the menu icons, not menu items. when I read about nav I found that "nav tag defines a set of navigation links". 
if it's wrong, please tell me the right definition.
Thanks

**Review by ahmetbozaci — APPROVED** [review]
# STATUS: Approved  🥳   🎉🎉🎉

### Hello @Ahmed-Alfaouq  ✋🏼 Nice work on the last review. Your project is great. I approve it.There is nothing else to say other than... it's time to merge it and delete the branch. You can submit this project in your dashboard.


### Optional suggestions

#### These are optional but highly recommended.

- [ ]  You didn't use same font-family from design. You can change it.
- [ ]  Instead of using pictures for icons you can use [font-awesome-icons](https://fontawesome.com/start). 

- [ ] For DRY css you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in html file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)


### Cheers and Happy coding! 💻 👏

___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**